src/main/java/laddergame/domain/Line.java

```diff
@@ -0,0 +1,32 @@
+package laddergame.domain;
+
+import java.util.List;
+import java.util.stream.IntStream;
+import laddergame.RandomGenerator;
+
+public class Line {
+
+    private final List<Boolean> points;
+    private final RandomGenerator randomGenerator = new RandomGenerator();
+
+    public Line(List<Boolean> points) {
+        validateLine(points);
+        this.points = points;
+    }
+
+    public Line(int count) {
+        List<Boolean> randomPoints = randomGenerator.generator(count);
+        validateLine(randomPoints);
+        this.points = randomPoints;
```

Comment:
```suggestion
        this(randomGenerator.generator(count));
```

주 생성자를 호출하지 않다보니 유효성 체크와 같은 중복 로직이 생긴다.
부 생성자가 주 생성자를 호출하도록 구현하는 습관을 들이면 어떨까?